ddtrace/llmobs/_integrations/gemini.py

```diff
@@ -51,6 +53,7 @@ def _llmobs_set_tags(
         if response is not None:
             output_messages = self._extract_output_message(response)
 
+        # extract metrcis below needs to be attached to the metrics thing here and then we are done
```

Comment:
```suggestion
        # extract metrics below needs to be attached to the metrics thing here and then we are done
```
Is this comment necessary? Or is this a temp wip comment